CardData.py

```diff
@@ -51,82 +51,13 @@
 INVALID_TOKENS = ['200175', '200176']
 
 """
- Categories are stored as sums of powers of 2. Use the ID from here to look up
- localisations in categories output file.
+ Use the ID from here to look up localisations in categories output file.
+ WARNING : this list is not simply 1 to 73, some numbers are missing
```

Comment:
Jamie, about your suggestion of moving this warning, I think this warning (rewritten) is better where the list would be changed, instead of where the list would be called.